ledger/accountdb.go

```diff
@@ -4433,7 +4508,8 @@ func (iterator *orderedAccountsIter) Next(ctx context.Context) (acct []accountAd
 			return nil
 		}
 
-		resCb := func(addr basics.Address, cidx basics.CreatableIndex, resData *resourcesData, encodedResourceData []byte) error {
+		// TODO verify if this needs to check resource limit as well
```

Comment:
please research this TODO - either implement the limiting logic, or explain why it is not needed here

Reply:
added logic